docs/macos_cheatsheet.md

```diff
@@ -2,10 +2,10 @@ Hello and welcome to this Apple M1 Cheatsheet.
 
 The goal is to gather here all tips and known issues about the new processor from Apple!
```

Comment:
If these are all issues pertaining to the m1 processor, then perhaps we just rename `macos_cheatsheet.md` to `apple_m1_cheatsheet.md`?

Otherwise, we change this here title to `... issues about with MacOs!`?

Reply:
They are all issues for macOS so we should rename the title indeed.